openassessment/xblock/static/js/src/lms/components/WaitingStepContent.jsx

```diff
@@ -55,17 +59,21 @@ const WaitingStepContent = ({ waitingStepDetails, refreshData }) => {
 WaitingStepContent.propTypes = {
   waitingStepDetails: PropTypes.shape({
     display_name: PropTypes.string,
-    must_be_graded_by: PropTypes.string,
-    must_grade: PropTypes.string,
-    waiting_count: PropTypes.string,
-    overwritten_count: PropTypes.string,
+    must_be_graded_by: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    must_grade: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    waiting_count: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    overwritten_count: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
[courious] why change this from PropTypes.string to PropTypes.onOfType?

Reply:
There is an error here, it watis an number but something recieves an string idk why that is happening